packages/components/molecules/f-alert/CHANGELOG.md

```diff
@@ -4,6 +4,17 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 
+v4.0.0
+------------------------------
+*November 26, 2021*
+
+### Added
+- **Breaking Change**: Added `f-button` dependency to peer dependencies. Now `f-button` should be included as a dependency of the consuming app.
```

Comment:
```suggestion
- **Breaking Change**: Added `f-button` dependency to peer dependencies. Now `f-button` should be included as a dependency of the consuming component or application.
```

Reply:
Is it worth adding something to the README? Maybe a section near the top that state's its dependencies?

Reply:
Yup i think that would be useful too.